### DISCUSSION TITLE
How to enable bash-completions?

### DISCUSSION THREAD

**Discussion by p3lim** [marlonrichert/zsh-autocomplete#686] [Q&A]
I'm running zsh-autocomplete from nixpkgs on Debian 12, and a bunch of (bash) completions for the system packages are not included.

E.g. `nix-shell --` will show arg completions, but `systemctl --` will not.

I've tried messing around with compinit/bashcompinit, and I've set the `skip_global_compinit=1` as suggested, but nothing seems to work.

**Comment by marlonrichert** [comment]
I looked at the source code for Debian's [`bash`](https://salsa.debian.org/debian/bash) and [`bash-completions`](https://salsa.debian.org/debian/bash-completion) packages and they don't include any completion for `systemctl`. Do you perhaps mean `sysctl`?
